examples/image_classification/cifar10_advanced.py

```diff
@@ -25,7 +25,7 @@
 from tqdm import tqdm, trange
 
 import objax
-from examples.classify.img.tfdata.data import DataSet
+from examples.image_classification.tfdata.data import DataSet
```

Comment:
Ideally I think we should remove `examples.image_classification.tfdata.data` completely and copy content of `DataSet` directly into this file.
The reason is because `examples.image_classification.tfdata.data` is only used in cifar10_advanced example, moreover it contains bunch of unused stuff. Thus presence of examples/image_classification/tfdata/data.py might add confusion for users

But it might be too much change for this pull request, so we can defer it for later.

Reply:
Agree in both principle and decision to leave for later.